pkg/providers/snow/reconciler/main_test.go

```diff
@@ -0,0 +1,14 @@
+package reconciler_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aws/eks-anywhere/internal/test/envtest"
+)
+
+var env *envtest.Environment
+
+func TestMain(m *testing.M) {
+	os.Exit(envtest.RunWithEnvironment(m, envtest.WithAssignment(&env)))
```

Comment:
We don't need this unless the tests use envtest. I think you are only using fake client, right?

Reply:
Without setting environment, fake client will not work, as it will refuse the objects we pass.

Reply:
Ahhh I see, we don't need init envtest, just import a package

try adding `_ "github.com/aws/eks-anywhere/internal/test/envtest"` to the imports

Reply:
Great, I'll make another PR for it